formats/json/commonMain/src/kotlinx/serialization/json/JsonConfiguration.kt

```diff
@@ -0,0 +1,36 @@
+package kotlinx.serialization.json
+
+import kotlinx.serialization.*
+import kotlinx.serialization.modules.*
+
+/**
+ * Configuration of the current [Json] instance available through [Json.configuration]
+ * and configured with [JsonBuilder] constructor.
+ *
+ * Can be used for debug purposes and for custom Json-specific serializers
+ * via [JsonEncoder] and [JsonDecoder].
+ *
+ * Standalone configuration object is meaningless and can nor be used outside of the
+ * [Json], neither new [Json] instance can be created from it.
+ */
+@ExperimentalSerializationApi
+public class JsonConfiguration internal constructor(
+    public val encodeDefaults: Boolean = false,
```

Comment:
I think we need to express somehow that documentation for this properties is available in JsonBuilder